src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -26,6 +29,8 @@
 using Newtonsoft.Json.Linq;
 using ProtoCore.Namespace;
 
+[assembly: InternalsVisibleTo("DynamoPackages")]
```

Comment:
another way to solve this is with reflection - if you go this route - others might use this unintentionally, please add a comment here at the minimum if possible.

Reply:
Comment added. I'm not sure what you mean here about solving with reflection. If you think that's a better strategy, can we discuss in person how to implement it?

Reply:
yes, let's talk about the different options we have besides this.